src/main/java/to/us/resume_builder/business/server_connect/request/BasicRequest.java

```diff
@@ -18,56 +18,58 @@
  */
 public abstract class BasicRequest<T> {
 
-    /** The client leveraged by all subclasses. */
-    protected static final HttpClient CLIENT;
-
-    /** Initializes the client */
-    static {
-        CLIENT = HttpClient.newBuilder().version(HttpClient.Version.HTTP_2).build();
-    }
-
-    /** Charset to send requests using */
-    protected static final String CHARSET = "UTF-8";
-    /** Separator for elements in a query string */
-    protected static final String SEPARATOR = "&";
-    /** Assignment for parameters in a query string */
-    protected static final String ASSIGN = "=";
-    /** Separates the URL from the query string */
-    protected static final String PARAM_SPLIT = "?";
-
-    /** Assumed timeout, in seconds, of the method */
-    protected static final int TIMEOUT = 15;
-
-    /** The type of request this handles */
-    private RequestType type;
-
-    /** The path to send the request to; includes hostname */
-    protected final String path;
-
-    /**
-     * Create a request with a specified type
-     * 
-     * @param type The type of request this request handles
-     * @param path The address to send the request to
-     */
-    public BasicRequest(RequestType type, String path) {
-        this.type = type;
-        this.path = path;
-    }
-
-    /**
-     * Creates and sends the specific request this class handles.
-     * <p>
-     * Example of Design Pattern: Template Method
-     * 
-     * @param arguments The arguments to send the request with, in a [Name1, Val1,
-     *                  Name2, Val2...] format.
-     * @return The http response from the request this class handles
-     * @throws InterruptedException If the operation is interrupted
-     * @throws IOException          If an I/O exception occurs in sending or
-     *                              receiving.
-     */
-    public final HttpResponse<T> sendRequest(String... arguments) throws IOException, InterruptedException {
+	/** The client leveraged by all subclasses. */
```

Comment:
Didn't realize that you were editing the BasicRequest class. I made some changes here in addition to fixing Javadoc compile errors (changing some stuff about this class that irked me), and it got onto master before this. I hope it doesn't make too painful a merge conflict to resolve.